_docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md

```diff
@@ -58,66 +54,64 @@ VMware Player or VMware Fusion:
               Log in to this virtual machine: Linux/Windows <Alt+F2>, Mac OS X <Options+F5>
 **Note:** The URL provided corresponds to the Web UI in Apache Drill.  
```

Comment:
Not translated. Keep consistent with the next NOTE text.

Reply:
Actually, I found that the "Note" should be part of MapR installation message.
So I keep both of the two "Note" in English.